docs/source/storeconv.md

```diff
@@ -115,56 +119,62 @@ be represented fully by splitting datasets in this way. In these cases:
 
 While an opener is free to define any open parameters for any of its datasets,
 there are some common parameters which are likely to be used by the majority
-of datasets. Also, there are some parameters which are fundamental for the 
+of datasets. Furthermore, there are some parameters which are fundamental for the 
 description of a dataset and therefore MUST be included in a schema 
 (these parameters are denoted explicitly in the list below). In case that an 
 opener does not support varying values of one of these parameters, a constant 
 value must defined. This may be achieved by the JSON schema's `const` property 
-or by an `enum` property whose value is a one-element array.
+or by an `enum` property value which is a one-element array.
```

Comment:
I find this less clear than the original. If you don't like "whose", I suggest:

> ... or by an `enum` property with a one-element array as its value.

Reply:
'Who is always associated with people. Which is used with things. ' https://www.grammarly.com/blog/when-to-use-which-and-who/
That's why I object to using 'whose' in this situation.

Reply:
That is not correct: "whose" does not necessarily refer to a "who" as a person, it may refer to this things as well (de: deren / dessen). See https://www.linguee.de/deutsch-englisch/search?source=auto&query=whose

Therefore please stay with the original.

Reply:
Please maintain word order: "... an enum property WHOSE value ..."

Reply:
"Who" is not the same as "whose". [Wikipedia has a whole article](https://en.wikipedia.org/wiki/Inanimate_whose) on "inanimate whose", and doesn't list a single modern style guide that recommends against it.

> *Merriam–Webster's Dictionary of English Usage* states that, amongst "the current books" that discussed the subject as of the late 20th century, "not one of them finds [inanimate] whose anything but standard". … It asserts that "notion that whose may not properly be used of anything except persons is a superstition" and such use is "entirely standard as an alternative to of which in all varieties of discourse".

[grammarly.com seems to feel the same way](https://www.grammarly.com/blog/whos-whose/):

> > Bessie carried a lantern, whose light glanced on wet steps and gravel road sodden by a recent thaw.

> This one’s worth an extra glance: “who” in all its forms generally refers to animate beings, but in the possessive there’s no equivalent for inanimate objects like Bessie’s lantern. The very awkward alternative is “Bessie carried a lantern, the light of which glanced on wet steps.” Not good.